components/director/cmd/director/main.go

```diff
@@ -437,11 +439,24 @@ func getTenantMappingHandlerFunc(transact persistence.Transactioner, authenticat
 	tenantConverter := tenant.NewConverter()
 	tenantRepo := tenant.NewRepository(tenantConverter)
 
+	consumerExistsCheckers := map[model.SystemAuthReferenceObjectType]func(ctx context.Context, id string) (bool, error){
```

Comment:
it's not the same - we have integration systems once, then runtimes, and we can also have applications - it's basically a wrapper of the service layer logic - instead of three different Svc fields in the CtxProvider we have this map

Reply:
Thanks for spotting this! 💯 As discussed, we shouldn't check the system auths, but the int systems